docs/_data/prague-2020-sessions.yaml

```diff
@@ -0,0 +1,627 @@
+    API examples for companies such as Oracle, Google, and Splunk, and I''ve learned
+    that good examples depend more on writing skills than coding skills. Based on
+    this knowledge, I''ll show you how to:</p>
+
+    <ul>
+
+    <li>Determine which examples you''ll need</li>
+
+    <li>Provide examples with the right amount of complexity</li>
+
+    <li>Work with SMEs to develop examples</li>
+
+    <li>Create style guides for writing examples </li>
+
+    <li>Store examples as live code and refer to them in documentation</li>
+
+    </ul>'
+- title: Emulating the Teacher's Approving Nod in Teaching Material
+  slug: emulating-the-teacher-s-approving-nod-in-teaching-material-tanks-transfeld
+  series: Write the Docs Prague
+  series_slug: emulating-the-teacher-s-approving-nod-in-teaching-material-tanks-transfeld
+  year: 2020
+  speakers:
+  - name: Tanks Transfeld
+    slug: tanks-transfeld
+    twitter: '@mirabellensaft'
+    website: 
+  abstract: '<p>While lecture style teaching barely has to be adapted when it moves
+    online, the story is different for student centered, interactive formats that
+    teach practical skills, like we do in our programming trainings. Compared to what
+    we are used to in a classroom, the different channels of communication are limited
+    in online platforms. A few examples: Nonverbal communication is extremely reduced.
+    Informal communication between participants, used to clarify on assignments or
+    information is much harder. If students are split up in groups to work on assignments,
+    there often is no channel to deliver additional information to all groups at once.
+    The fact that the participants of our embedded trainings have to interact with
+    microcontrollers brings further challenges: assisting students with debugging
+    hardware can only be done verbally as we have no possibility to physically check
+    student''s hardware, if needed.</p>
+
+    <p>When we had to move our face-to-face trainings online because of the corona
+    virus crisis, this is how we addressed the lack of personal interaction in the
+    design of our material:</p>
+
+    <ul>
+
+    <li>
+
+    <p>We provided a walkthrough with </p>
+
+    <ul>
+
+    <li>explicitly phrased tasks</li>
+
+    <li>expected outputs</li>
+
+    <li>troubleshooting pages</li>
+
+    <li>problem solving strategies and tiered aids</li>
+
+    <li>answers to expected questions</li>
+
+    <li>additional explanations</li>
+
+    </ul>
+
+    </li>
+
+    <li>
+
+    <p>To structure this huge amount of information we</p>
+
+    <ul>
+
+    <li>used a book structure with chapters</li>
+
+    <li>visually marked different kinds of information (action items, more details,
+    etc)</li>
+
+    <li>separated assignments from solving strategies</li>
+
+    </ul>
+
+    </li>
+
+    <li>
+
+    <p>We debugged our docs continously before deployment by</p>
```

Comment:
These <p> tags are superfluous and create line breaks. Don't know how easy it is to fix in their pretalx abstract?

Reply:
Yeah, this is partly a problem with our CSS though.